### DISCUSSION TITLE
Add `network_tests/` outside of package folder

### DISCUSSION THREAD

**Discussion by jbischof** [keras-team/keras-hub#306] [Ideas]
We are adding pretrained tokenizers (PR #287) and models (PR #299) to the package in recent PRs. While we will want to periodically verify the the necessary files are available and have the right permissions, we cannot create unit tests with large network calls (especially when models are as much as 1GB).

I propose to add an `network_tests/` or similarly named folder outside the `keras_nlp` package folder where we can add tests for loading pretrained objects that can be run manually as needed with `pytest network_tests/`.

**Comment by chenmoneygithub** [comment]
Sounds good, one question:

Will this be more like a benchmarking or unit testing? For benchmarking, we probably want to do periodic run on GCP instead of pytest. Or maybe we want both?

**Comment by jbischof** [comment]
My initial plan was just to test that the files are available and have the right permissions. This only needs to be done on occasion. Would the same be true for the benchmarks or do you want to run them manually @chenmoneygithub?
